frontend/sw360-portlet/src/main/webapp/html/components/includes/releases/vulnerabilities.jspf

```diff
@@ -70,7 +70,7 @@
                 '<select class="toplabelledInput" ' +
                 'id="verificationChangeSelectFor<sw360:out value='${vulnerability.externalId}'/>" ' +
                 'name="<portlet:namespace/><%=VerificationState.class%>" ' +
-                'onchange="changeVerification(this, `<sw360:out value='${vulnerability.externalId}'/>`,`${vulnerabilityVerifications.get(vulnerability.externalId).get(releaseId).ordinal()}`)" ' +
+                "onchange='changeVerification(this, &quot;<sw360:out value='${vulnerability.externalId}'/>&quot;,&quot;${vulnerabilityVerifications.get(vulnerability.externalId).get(releaseId).ordinal()}&quot;)' " +
```

Comment:
Are you sure that this works? I would expect escaped quotes, e.g. `\"`, instead of `&quot;` which is html escaping within pure JS?